recipes/cgat-apps/meta.yaml

```diff
@@ -32,31 +33,29 @@ requirements:
   run:
     - python
     - cgatcore
-    - alignlib-lite 
-    - biopython 
-    - cython 0.29.*
-    - future 
-    - matplotlib 
-    - numpy 
-    - pandas 
-    - pybedtools 
-    - pybigwig 
-    - pysam 
-    - python-lzo 
-    - pyyaml 
+    - alignlib-lite
+    - biopython
+    - future
+    - matplotlib
+    - {{ pin_compatible('numpy') }}
+    - pandas
+    - pybedtools
+    - pybigwig
+    - {{ pin_compatible('pysam') }}
+    - python-lzo
+    - pyyaml
+    - quicksect
     - scipy
-    - setuptools
     - six 
+    - sortedcontainers
     # Misc dependencies
     - bedtools
     - ucsc-bedgraphtobigwig
     - ucsc-bedtobigbed
     - ucsc-wigtobigwig
-    - coreutils 
-    - grep 
-    - libpng 
-    - zlib
-    - quicksect
+    - coreutils
+    - grep
+    - {{ pin_compatible('libpng') }}
```

Comment:
The pin_compatible stuff is already done for libpng (via conda-forge-pinning), you don't need to do it explicitly.